src/pages/_app.tsx

```diff
@@ -47,7 +47,7 @@ export default function MyApp(props: MyAppProps) {
             <RelayApiContextProvider>
               <UseInkathonProvider
                 appName='Coretime UI'
-                connectOnInit={true}
+                connectOnInit={false}
```

Comment:
I think it is a much better experience to connect on init. It is a bit annoying to always have to manually connect the wallet.